protocol/v2/ssv/validator/duty_executer.go

```diff
@@ -34,6 +34,8 @@ func (c *Committee) OnExecuteDuty(logger *zap.Logger, msg *types.EventMsg) error
 		return fmt.Errorf("failed to get execute committee duty data: %w", err)
 	}
 
+	logger = logger.With(fields.DutyID(fields.FormatCommitteeDutyID(c.Operator.Committee, c.BeaconNetwork.EstimatedEpochAtSlot(executeDutyData.Duty.Slot), executeDutyData.Duty.Slot)))
```

Comment:
won't this only show the duty ID in the duty start flow? what about the flow of processing incoming messages? 

do you see this field in "got commit quorum" log, for example?

Reply:
@moshe-blox, I've just tested this on stage. It works fine. I see logs for `"got commit quorum"`

Reply:
As we pass the logger with `duty_id` field added into ConsumeQueue -> ProcessMessage, we also have in `ProcessConsensus` and `ProcessPostConsensus` as well